### PR TITLE
[Conflict Resolver] update-react-table | Unresolved Conflicts

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

**Comment by kongtiaowang** [comment]
@alisterdev 
Could you review this?

File: modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc [hunk 1]
```diff
@@ -206,20 +206,23 @@ class NDB_Menu_Filter_Form_Conflict_Resolver extends NDB_Menu_Filter_Form
...
         // set the class variables
         $this->columns = array(
-                          'conflicts_unresolved.TableName',
-                          'session.CandID',
-                          'candidate.PSCID',
-                          'session.Visit_label',
+                          'conflicts_unresolved.TableName as Instrument',
+                          'session.CandID as CandID',
+                          'candidate.PSCID as PSCID',
+                          'session.Visit_label as Visit_Lable',
```
Comment (alisterdev) [inline modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc]: Typo in Visit_Label

Comment (alisterdev) [inline modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc]: As a matter of fact, none of the aliases assigned in this piece of code are used anywhere, so there's no reason to add them

File: modules/conflict_resolver/templates/menu_conflict_resolver.tpl [hunk 2]
```diff
@@ -78,95 +78,22 @@
...
-        location.href="{$baseurl}/conflict_resolver/?filter[order][field]={$filterfield}&filter[order][fieldOrder]={$filterfieldOrder}&pageID=" + pageNum
-    },
-    Active: {$pageID}
-});
-React.render(pageLinks, document.getElementById("pageLinks"));
+loris.hiddenHeaders = {(empty($hiddenHeaders))? [] : $hiddenHeaders };
+var table = RDynamicDataTable({
+     "DataURL" : "{$baseurl}/conflict_resolver/?format=json",
+     "getFormattedCell" : formatColumn,
+     "freezeColumn" : "PSCID"
```
Comment (alisterdev) [inline modules/conflict_resolver/templates/menu_conflict_resolver.tpl]: The 'Instrument' should be the frozen column

File: modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc (hunk 1 above)
Comment (kongtiaowang) [inline modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc]: @alisterdev 
Doing that, it is easy to match columns and headers. like a comment.

File: modules/conflict_resolver/templates/menu_conflict_resolver.tpl (hunk 2 above)
Comment (kongtiaowang) [inline modules/conflict_resolver/templates/menu_conflict_resolver.tpl]: @alisterdev 
all fixed.

**Comment by alisterdev** [comment]
Nevermind, found a bug -,-

**Pull request merged**